extract/tunnel-example.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Copy this script, change the remote user and host,
+# then run it to open a local tunnel to the remote MongoDB service.
+
+##
+## DO NOT COMMIT YOUR COPY OF THIS FILE TO THE REPOSITORY
+##
+
+ssh -Nv -L127.0.0.1:27017:0.0.0.0:27017 remote-user@remote-host.example.org
```

Comment:
I'll add that.  I have several `IdentityFile` directives in my `~/.ssh/config` which specify the various PEM files to be used.  I believe that's as according to our wiki.  However, some may forget to set that up, so I'll add something about it to the project.  The problem is, not every environment will require the PEM(s) and different environments require different PEM(s).  (For example, we have four: one for each OpenLRS app server and DB server, production and development.)

New issue: https://itsjira.umms.med.umich.edu/browse/TLCALIPER-106